internal/command/postgres/backup.go

```diff
@@ -468,7 +426,7 @@ func resolveRestoreTarget(ctx context.Context) string {
 	return target
 }
 
-func hasRequiredVersion(appName string, machines []*fly.Machine) error {
+func hasRequiredVersionForBackup(appName string, machines []*fly.Machine) error {
```

Comment:
Can we do the same thing here that we did below?  Define the version as a constant and just reuse `hasRequiredVersiononMachines` ?